libs/recnet-api-model/src/lib/api/user.ts

```diff
@@ -31,7 +31,13 @@ export const getUserMeResponseSchema = z.object({
 export type GetUserMeResponse = z.infer<typeof getUserMeResponseSchema>;
 
 // PATCH /users/me
-export const patchUserMeRequestSchema = userSchema.partial();
+export const patchUserMeRequestSchema = userSchema
+  .omit({
+    id: true,
+    numFollowers: true,
+    following: true,
+  })
```

Comment:
I removed the role but retained the email field, because I feel like it is a good feature for user to change their email and maybe we can do that in the future.